fdbserver/Status.actor.cpp

```diff
@@ -1122,33 +1122,66 @@ ACTOR static Future<JsonBuilderObject> latencyProbeFetcher(Database cx, JsonBuil
 	return statusObj;
 }
 
-ACTOR static Future<Void> consistencyCheckStatusFetcher(Database cx, JsonBuilderArray *messages, std::set<std::string> *incomplete_reasons, bool isAvailable) {
-	if(isAvailable) {
-		try {
-			state Transaction tr(cx);
-			loop {
-				try {
-					tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
-					tr.setOption(FDBTransactionOptions::LOCK_AWARE);
-					tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
-					Optional<Value> ccSuspendVal = wait(timeoutError(BUGGIFY ? Never() : tr.get(fdbShouldConsistencyCheckBeSuspended), 5.0));
-					bool ccSuspend = ccSuspendVal.present() ? BinaryReader::fromStringRef<bool>(ccSuspendVal.get(), Unversioned()) : false;
-					if(ccSuspend) {
-						messages->push_back(JsonString::makeMessage("consistencycheck_disabled", "Consistency checker is disabled."));
-					}
+ACTOR static Future<Void> consistencyCheckStatusFetcher(Database cx, JsonBuilderArray *messages, std::set<std::string> *incomplete_reasons) {
+	try {
+		state Transaction tr(cx);
+		loop {
+			try {
+				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+				tr.setOption(FDBTransactionOptions::LOCK_AWARE);
+				tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+				Optional<Value> ccSuspendVal = wait(timeoutError(BUGGIFY ? Never() : tr.get(fdbShouldConsistencyCheckBeSuspended), 5.0));
+				bool ccSuspend = ccSuspendVal.present() ? BinaryReader::fromStringRef<bool>(ccSuspendVal.get(), Unversioned()) : false;
+				if(ccSuspend) {
+					messages->push_back(JsonString::makeMessage("consistencycheck_disabled", "Consistency checker is disabled."));
+				}
+				break;
+			} catch(Error &e) {
+				if(e.code() == error_code_timed_out) {
+					messages->push_back(JsonString::makeMessage("consistencycheck_suspendkey_fetch_timeout",
+						format("Timed out trying to fetch `%s` from the database.", printable(fdbShouldConsistencyCheckBeSuspended).c_str()).c_str()));
 					break;
-				} catch(Error &e) {
-					if(e.code() == error_code_timed_out) {
-						messages->push_back(JsonString::makeMessage("consistencycheck_suspendkey_fetch_timeout",
-							format("Timed out trying to fetch `%s` from the database.", printable(fdbShouldConsistencyCheckBeSuspended).c_str()).c_str()));
+				}
+				wait(tr.onError(e));
+			}
+		}
+	} catch(Error &e) {
+		incomplete_reasons->insert(format("Unable to retrieve consistency check settings (%s).", e.what()));
+	}
+	return Void();
+}
+
+ACTOR static Future<Void> logRangeWarningFetcher(Database cx, JsonBuilderArray *messages, std::set<std::string> *incomplete_reasons) {
+	try {
+		state Transaction tr(cx);
+		loop {
+			try {
+				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+				tr.setOption(FDBTransactionOptions::LOCK_AWARE);
```

Comment:
I generally push for using the minimal required options for these (though I know that hasn't been consistently done in other checks here):

```suggestion
				tr.setOption(FDBTransactionOptions::READ_LOCK_AWARE);
```